bot/internal/review/review.go

```diff
@@ -168,13 +168,13 @@ func (r *Assignments) Get(e *env.Environment, docs bool, code bool, files []gith
 	return reviewers
 }
 
-func (r *Assignments) getDocsReviewers(author string) []string {
-	setA, setB := getReviewerSets(author, "Core", r.c.DocsReviewers, r.c.DocsReviewersOmit)
+func (r *Assignments) getDocsReviewers(e *env.Environment) []string {
+	setA, setB := getReviewerSets(e, "Core", r.c.DocsReviewers, r.c.DocsReviewersOmit)
```

Comment:
Is the team always supposed to be "Core" here?

Reply:
Sort of, the logic for handling docs Pr specifically only really applies to core.

It's something we've purposefully ignored/avoided for the cloud repo